s3.tf

```diff
@@ -0,0 +1,30 @@
+module "s3_log_label" {
+  source    = "github.com/cloudposse/tf_label"
+  namespace = "${var.namespace}"
+  stage     = "${var.stage}"
+  name      = "${var.s3_bucket_efs_name}"
```

Comment:
I don't understand the `s3_bucket_efs_name` variable name. Is it a bucket or efs? ;)